docs/readmes/lte/configure_sentry.md

```diff
@@ -44,6 +44,28 @@ sentry_upload_mme_log: false
 sentry_sample_rate: 1.0
 ```
 
+## Client-side filters for error messages
+
+Another configuration option is a list of error messages that should be filtered and not be sent to Sentry. This is done to avoid spamming Sentry with errors that can occur frequently but that are not bugs to be monitored.
```

Comment:
Maybe we can say that this is similar in concept to the server side fliter configuration made available by Sentry.io: https://docs.sentry.io/product/accounts/quotas/#inbound-data-filters

Reply:
Added some information about the server-side filters.